sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -332,6 +332,13 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
       s"If necessary set ${SQLConf.ANSI_ENABLED.key} to false to bypass this error.", e)
   }
 
+  def illegalUrlError(url: UTF8String, e: IllegalArgumentException):
+  Throwable with SparkThrowable = {
+    new SparkIllegalArgumentException(errorClass = "CANNOT_DECODE_URL",
+      messageParameters = Array(url.toString, e.getMessage)
```

Comment:
We should avoid getting the error message from the nested Java exceptions. For example, if we will translate the error messages in error-classes.json to a local language, your error message will have mixed text in at least 2 languages. There is a mechanism to provide more details via sub-classes but so far let's remove  `<details>`.

Reply:
ok. Let's leave this as is.